app/templates/codelijsten.hbs

```diff
@@ -64,13 +67,13 @@
               @size="small"
             >
               <ul class="au-u-margin-bottom-small">
-                <li>Kopieer de link en plak ze bij
+                <li>Copy the link and past it at
                   <strong>form:options</strong>
-                  en vervang de link achter
+                  and replace the link behind
                   <strong>conceptScheme</strong>
-                  om de concept scheme te activeren.</li>
-                <li>Sommige links omvatten meer informatie over de concept
-                  scheme zelf. Klik op de link om die informatie te achterhalen.</li>
+                  to activate the concept scheme</li>
+                <li>Some of the links contain more information about the concept
+                  scheme itself. Click the link top get the information</li>
```

Comment:
Didn't put this in the translation file yet. As it has some styling applied to it. Plus possibly this page will be replaced with the new code lists view from #84